bitnami/rabbitmq/values.yaml

```
@@ -545,6 +561,37 @@ advancedConfigurationExistingSecret: ""
 ## @param featureFlags that controls what features are considered to be enabled or available on all cluster nodes.
 ##
 featureFlags: ""
+
+datadog:
+  enabled: true
+  #max_detailed_queues: 200
+  annotations:
+    ad.datadoghq.com/rabbitmq.check_names: '["rabbitmq"]'
+    ad.datadoghq.com/rabbitmq.init_configs: '[{}]'
+    ad.datadoghq.com/rabbitmq.instances: >-
+      [{
+       {{- if .Values.datadog.max_detailed_queues }}
+        "max_detailed_queues": {{ .Values.datadog.max_detailed_queues | quote }},
+        {{- end }}
+        "rabbitmq_api_url":"http://%%host%%:15672/api/",
+        "username":"datadog",
+        "password":"%%env_RABBITMQ_DATADOG_PASSWORD%%"
+      }]
+    ad.datadoghq.com/rabbitmq.logs: >-
+      [{
+        "source": "rabbitmq",
+        "service": "{{ .Release.Name }}",
+        "log_processing_rules": [
+          {
+            "type": "multi_line",
+            "name": "logs_starts_with_equal_sign",
+            "pattern" : "="
+          }
+        ]
+      }]
+  labels:
+    team: "team_dbre" # name of the team ex:team_dbre, used for cost control
```

Let's apply the new naming convention (& improve the comment):
```suggestion
    team: "dbre"   # should *not* be overwritten in HR, because DBRE is maintaining the pods & should receive alerts about them
```